README.md

```diff
@@ -45,9 +45,9 @@ Ruby on Rails, Visual Studia Code, WSL
 
 - [Live Demo Link]()
```

Comment:
- Kindly, having the online version is not optional, please fix what the previous reviewer suggested and add it to fit the requirement and have better user accessibility :+1:  
![2024-01-27_09-47](https://github.com/Uzair-Manzoor/budget_app/assets/52010789/917d352d-4f10-47b0-958f-03740f19c9ef)